src/leds.c

```diff
@@ -40,6 +40,7 @@
 #define W(c) ((uint8_t) ((c) >> 24))
 #define RGB(r, g, b) ((r) << 16 | (g) << 8 | (b))
 #define RGBW(r, g, b, w) ((w) << 24 | (r) << 16 | (g) << 8 | (b))
+#define TAU (2.0 * acosf(-1.0))
```

Comment:
That's a very creative and complex way to declare a `2 * PI` constant :grinning:. I don't even think the compiler would optimize this, so the `acosf()` would be called on every occurrence too...

Reply:
True, at this point I was just writing code to get the colours looking good, I was beyond caring about details :joy: I'll redefine this as `2 * M_PI`